app/controllers/orders_controller.rb

```diff
@@ -13,56 +14,110 @@ def index
   def show
     logger.tagged("#{current_user.urs_uid} #{controller_name}_controller") do
       @order = Echo::Order.new(client: echo_client, echo_provider_token: echo_provider_token, guid: params['id'])
-
       add_breadcrumb @order.guid, order_path(@order.guid)
-
       render :show
     end
   end
 
   def search
-    logger.tagged("#{current_user.urs_uid} #{controller_name}_controller") do
-      @orders = Echo::Orders.new(client: echo_client, echo_provider_token: echo_provider_token, guids: determine_order_guids).orders
+    Rails.logger.info("starting request - #{request.uuid} timeout=#{echo_client.timeout}")
+    init_time_tracking_variables
+    guids = determine_order_guids
```

Comment:
if you run `determine_order_guids` outside of the `logger.tagged` block, it won't have those logging tags.

Reply:
I fixed it, thanks.